src/paperoni/discovery/openalex.py

```diff
@@ -341,13 +341,21 @@ def query(
                             author=name,
                             institution=institution,
                             title=title,
+                            page=page,
+                            per_page=per_page,
+                            limit=limit,
+                            verbose=verbose,
```

Comment:
It seams to me that these args should be present for the behaviour to be what I'm currently expecting but I might be wrong

Reply:
I've removed limit but kept `page` and `per_page` args forwarding as well as printing a warning message. Let me know what you prefer